CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # master
 
+* Add support for ERB templates as preview examples.
```

Comment:
Do we want to support all template handlers, not just ERB?

Reply:
Yeah, given we are just calling render in the controller I think we are good to say that we support any handle, no?

Reply:
Might we update this message to reflect that all handlers are supported?